tests/testassertions.py

```diff
@@ -135,6 +135,12 @@ def assertRanInitialClone(self, logpath, loglines):
         self._find(logpath, loglines,
                    self.initial_clone_command, self.update_cache_command)
 
+    def assertSSLVerifyFalse(self, logpath, loglines):
+        self._find(logpath, loglines,
+                   self.initial_clone_command +
+                   '.*' + self.sslverify_false_args,
+                   self.sslverify_false_args + 'true')
```

Comment:
This value isn't very meaningful but I recognise it's a symptom of `_find()` assuming that you want to check for something not being there, and in this case we don't.